src/front/wgsl/mod.rs

```diff
@@ -78,16 +116,16 @@ impl<'a> Error<'a> {
 
 #[derive(Clone, Debug, Error)]
 pub enum Error<'a> {
-    #[error("unexpected token {:?}, expected {1}", (.0).0)]
+    #[error("")]
     Unexpected(TokenSpan<'a>, &'a str),
-    #[error("unable to parse `{0}` as integer: {1}")]
-    BadInteger(&'a str, std::num::ParseIntError),
-    #[error("unable to parse `{1}` as float: {1}")]
-    BadFloat(&'a str, std::num::ParseFloatError),
-    #[error("unable to parse `{0}{1}{2}` as scalar width: {3}")]
-    BadScalarWidth(&'a str, char, &'a str, std::num::ParseIntError),
-    #[error("bad field accessor `{0}`")]
-    BadAccessor(&'a str),
+    #[error("")]
```

Comment:
Do we even need to implement `Error` here?

Reply:
Yeah this is just temporary. If we don't have a full error message implemented for either of these variants, we currently fall back to the messages listed here. We can remove `Error` here once they've all been updated.